Snippets/Snippets.json

```diff
@@ -24,47 +24,71 @@
-        "# Sentinel - Set up environment for msticpy         #",
+        "# Sentinel - Set up environment for MSTICPy         #",
         "#####################################################",
         "from pathlib import Path",
-        "import os",
-        "import sys",
-        "import warnings",
-        "from IPython.display import display, HTML, Markdown",
+        "from IPython.display import display, HTML",
+        "",
+        "update_nbcheck = (",
+        "    '<p style='color: orange; text-align=left'>'",
+        "    '<b>Warning: we needed to update '<i>utils/nb_check.py</i>'</b><br>'",
+        "    'Please restart the kernel and re-run this cell.'",
+        "    '</p>')",
+        "",
+        "REQ_PYTHON_VER= '3.6'",
+        "REQ_MSTICPY_VER='1.2.3'",
         "",
-        "REQ_PYTHON_VER=(3, 6)",
-        "REQ_MSTICPY_VER=(0, 6, 0)",
+        "display(HTML('Checking upgrade to latest msticpy version'))",
+        "%pip install --upgrade --quiet msticpy[azuresentinel]",
         "",
         "display(HTML('<h3>Starting Notebook setup...</h3>'))",
         "# If you did not clone the entire Azure-Sentinel-Notebooks repo you may not have this file",
         "if Path('./utils/nb_check.py').is_file():",
-        "    from utils.nb_check import check_python_ver, check_mp_ver",
-        "",
-        "    check_python_ver(min_py_ver=REQ_PYTHON_VER)",
         "    try:",
-        "        check_mp_ver(min_msticpy_ver=REQ_MSTICPY_VER)",
-        "    except ImportError:",
-        "        !pip install --upgrade msticpy",
-        "        if 'msticpy' in sys.modules:",
-        "            importlib.reload(sys.modules['msticpy'])",
-        "        else:",
-        "            import msticpy",
-        "        check_mp_ver(REQ_MSTICPY_VER)",
+        "        from utils.nb_check import check_versions",
+        "    except ImportError as err:",
+        "       !curl https://raw.githubusercontent.com/Azure/Azure-Sentinel-Notebooks/master/utils/nb_check.py > ./utils/nb_check.py 2>/dev/null",
+        "       display(HTML(update_nbcheck))",
+        "    if 'check_versions' not in globals():",
+        "        raise ImportError('Old version of nb_check.py detected - see instructions below')",
+        "    check_versions(REQ_PYTHON_VER, REQ_MSTICPY_VER)",
+        "",
         "",
         "from msticpy.nbtools import nbinit",
         "nbinit.init_notebook(",
         "    namespace=globals(),",
-        "    extra_imports=['ipwhois, IPWhois', 'urllib.request, urlretrieve', 'yaml']",
+        "    extra_imports=[]",
         ")"
       ],
-      "description": "Set up the execution environment for msticpy"
+      "description": "Set up the execution environment for MSTICPy"
     },
   
+    "Sentinel - Connect with MSTICPy": {
+      "prefix": ["msticpy-connect"],
+      "body": [
+        "######################################",
+        "# Sentinel -Connect with MSTICPy     #",
```

Comment:
space after "-"
Also maybe we should change "connect.." to "connect/authenticate..."